types/node/test/fs.ts

```diff
@@ -369,6 +369,16 @@ async () => {
     })).buffer; // $ExpectType Uint32Array
 };
 
+{
+    fs.createWriteStream('./index.d.ts');
```

Comment:
We should add a failing test here using `// $ExpectError`, these tests would pass even without your changes.